test/e2e/apimachinery/webhook.go

```diff
@@ -52,15 +52,16 @@ const (
 	roleBindingName = "webhook-auth-reader"
 
 	// The webhook configuration names should not be reused between test instances.
-	crdWebhookConfigName         = "e2e-test-webhook-config-crd"
+	crWebhookConfigName          = "e2e-test-webhook-config-cr"
```

Comment:
Need another rebase to dissolve these changes.

Reply:
done